spec/javascripts/app/views/location_stream_spec.js

```diff
@@ -0,0 +1,39 @@
+describe("app.views.LocationMap", function() {
+  beforeEach(function(){
+    this.post = factory.post();
+    this.view = new app.views.LocationStream({model : this.post});
+  });
+
+  describe("toggleMap", function() {
+    context("with location provided", function() {
+      beforeEach(function(){
+        this.post.set({location : factory.location()}); // set location
+        this.view.render();
+        console.log(this.view.$el.find(".mapContainer")[0]);
+      });
+
+      it("should contain a map container", function() {
+        expect(this.view.$el[0]).toContainElement(".mapContainer");
+      });
+
+      it("should initialize map", function() {
+        expect(this.view.$el.find(".mapContainer")[0]).toHaveClass("empty");
+        this.view.toggleMap();
+        expect(this.view.$el.find(".mapContainer")[0]).not.toHaveClass("empty");
+      });
+      /*
+       * does not work .. not sure why
```

Comment:
we have problems to get that test running. The function call toggleMap seems not do do anything inside this particular it-block.